lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -83,10 +89,23 @@
                     var info = {
                         alreadyMember: false,
                         memberOfCurrentTeam: false,
-                        teamHasSpace: false
+                        teamHasSpace: false,
+                        isAdminOrStaff: false,
+                        isInstructorManagedTopic: false
                     };
 
                     info.memberOfCurrentTeam = TeamUtils.isUserMemberOfTeam(this.model.get('membership'), username);
+                    info.isAdminOrStaff = this.context.userInfo.privileged || this.context.userInfo.staff;
+
+                    var topic_id = this.model.get('topic_id');
```

Comment:
This logic is repeated again. Could you please refactor the next 10 lines into a function into the `lms/djangoapps/teams/static/teams/js/views/team_utils.js` file? This way, you only need to write these once in javascript